src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -3874,4 +3930,35 @@ int IDebugSettingsCallback110.ShouldSuppressImplicitToStringCalls(out int pfSupp
             }
         }
     }
+
+    internal class SendInvalidateRequest : DebugRequestWithResponse<SendInvalidateArguments, SendInvalidateResponse>
+    {
+ 
+        public SendInvalidateRequest(): base("sendInvalidate")
+        {
+        }
+    }
+
+    internal class SendInvalidateResponse:ResponseBody
+    {
+        public sealed class Body
+        {
+            [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
Can we define all the fields here instead of creating a json string? Let the JSON librarly handle the serialize of the class.

Reply:
Removed all JsonProperty